models/demos/deepseek_v3/tests/test_mla_1d.py

```diff
@@ -79,6 +84,10 @@ def get_cache_on_host(tt_cache: ttnn.Tensor, row_idx: int, mesh_device: ttnn.Mes
     ],
     indirect=True,
 )
+@pytest.mark.parametrize(
```

Comment:
Like in the other tests, instead of having 2 arguments (weights_type) determine whether to use random or real weights by making this argument None for random tests